pkg/rebuild/rebuild/compare.go

```diff
@@ -29,6 +30,8 @@ func UpstreamArtifactReader(ctx context.Context, t Target, mux RegistryMux) (io.
 			return nil, errors.Errorf("failed to parse debian component: %s", t.Package)
 		}
 		return mux.Debian.Artifact(ctx, component, name, t.Artifact)
+	case Maven:
+		return mux.Maven.ReleaseFile(ctx, t.Package, t.Version, maven.TypeJar)
```

Comment:
@msuozzo @wbxyz do you think we should have another API `Artifact` so that we can make this generic and not just for `.jar` files? We could also extract artifact type from target here if we don't want to create another API.